separatemodels/src/test/java/pl/altkom/asc/lab/cqrs/intro/separatemodels/readmodel/PolicyInfoDtoFinderTest.java

```diff
@@ -0,0 +1,139 @@
+package pl.altkom.asc.lab.cqrs.intro.separatemodels.readmodel;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+import pl.altkom.asc.lab.cqrs.intro.separatemodels.SeparateModelsApplication;
+import pl.altkom.asc.lab.cqrs.intro.separatemodels.domain.Policy;
+import pl.altkom.asc.lab.cqrs.intro.separatemodels.testdatabuilders.CarsTestDataBuilder;
+import pl.altkom.asc.lab.cqrs.intro.separatemodels.testdatabuilders.PersonsTestDataBuilder;
+import pl.altkom.asc.lab.cqrs.intro.separatemodels.testdatabuilders.PoliciesTestDataBuilder;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@RunWith(SpringRunner.class)
+@ContextConfiguration(classes = SeparateModelsApplication.class)
+public class PolicyInfoDtoFinderTest {
+
+    private static final String POLICY_NUMBER_POL_0001 = "POL0001";
+    private static final String POLICY_NUMBER_POL_0002 = "POL0002";
+    @Autowired
+    PolicyInfoDtoProjection policyInfoDtoProjection;
+
+    @Autowired
+    PolicyInfoDtoFinder policyInfoDtoFinder;
+
+    @Autowired
+    JdbcTemplate jdbcTemplate;
+
+    @Before
+    public void cleanTable() {
+        jdbcTemplate.update("DELETE FROM policy_info_dto");
+        Policy pol0001 = PoliciesTestDataBuilder.standardOneYearPolicy(LocalDate.of(2019, 1, 1), POLICY_NUMBER_POL_0001, PersonsTestDataBuilder.kowalski(), CarsTestDataBuilder.oldFordFocus());
+        policyInfoDtoProjection.createPolicyInfoDto(pol0001);
+        Policy pol0002 = PoliciesTestDataBuilder.standardOneYearPolicy(LocalDate.of(2020, 2, 3), POLICY_NUMBER_POL_0002, PersonsTestDataBuilder.nowak(), CarsTestDataBuilder.oldSkodaFabia());
+        policyInfoDtoProjection.createPolicyInfoDto(pol0002);
+    }
+
+    @Test
+    public void should_return_two_polices_when_no_filter_condition() {
```

Comment:
I think the project uses different test naming convention - camel case instead of underscores...